lib/utils/image_to_text.rb

```diff
@@ -127,6 +143,8 @@ def extract_text_from_page(page)
   end
 
   def extract_text_with_tesseract(page)
+    # return nil if we can not find tessaract binary
+    return nil if !Discourse::Utils.which("tesseract")
```

Comment:
```suggestion
    return nil if !self.class.tesseract_installed?
```

I think we can just use the new method you defined in this PR. Also not sure if `Discourse::Utils.which` is valid because I can't find the method definition anywhere.

Reply:
yeah good catch, oops